src/core/env/ClientContext.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import BrowserApiClient from 'core/api/client/BrowserApiClient';
+import CssBaseline from '@mui/material/CssBaseline';
+import Environment from 'core/env/Environment';
+import { EnvProvider } from 'core/env/EnvContext';
+import { IntlProvider } from 'react-intl';
+import { MessageList } from 'utils/locale';
+import { Provider as ReduxProvider } from 'react-redux';
+import { store } from 'core/store';
+import { themeWithLocale } from '../../theme';
+import { FC, ReactNode } from 'react';
+import {
+  StyledEngineProvider,
+  Theme,
+  ThemeProvider,
+} from '@mui/material/styles';
+
+declare module '@mui/styles/defaultTheme' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface DefaultTheme extends Theme {}
+}
+
+export type ClientContextProps = {
```

Comment:
Does this need to be exported?

Reply:
Removed! I have a defensive coding habit of always exporting prop typedefs in case anything suddenly wants to import them. Doesn't seem to be the normal thing here!